src/hotspot/share/runtime/continuationJavaClasses.inline.hpp

```diff
@@ -185,4 +185,20 @@ inline void jdk_internal_vm_StackChunk::set_maxThawingSize(oop chunk, int value)
   chunk->int_field_put(_maxThawingSize_offset, value);
 }
 
+inline uint8_t jdk_internal_vm_StackChunk::lockStackSize(oop chunk) {
+  return Atomic::load(chunk->field_addr<uint8_t>(_lockStackSize_offset));
```

Comment:
If these accesses need to be atomic, could you add a comment explaining why?

Reply:
It is read concurrently by GC threads. Added comment.